src/app/components/HoldConfirmation/HoldConfirmation.jsx

```diff
@@ -116,7 +117,7 @@ class HoldConfirmation extends React.Component {
 
     return (
       <div id="mainContent">
-        <div className="page-header">
+        <div className="nypl-request-page-header">
```

Comment:
Can you add the `<h1>` here so it can display "Research Discovery (beta)"? The class for this should temporarily be "nypl-homepage-hero" just for consistency and for the demo.